.eslintrc

```diff
@@ -104,7 +104,7 @@
     "no-sync": 0,
     "no-empty-label": 2,
     "no-unreachable": 2,
-    "no-eval": 2,
+    "no-eval": 0,
```

Comment:
Not sure if we can disable this just for the tests, it maybe would be better to use comments to disable this linting error.